.github/workflows/helm-validation.yml

```diff
@@ -72,6 +72,14 @@ jobs:
       run: |
         .github/scripts/verify_helm_install.sh
 
+    # the script accepts number of requests for  APIs for CREATE_BUCKET, DELETE_BUCKET, GET_INFO
```

Comment:
same comment as previous file

Reply:
changed in https://github.com/scality/cosi-driver/pull/83/commits/8cc730891878aa9154d6d951db8032698b2aa2e5
Thanks